packages/react-native/src/theme/createTheme.ts

```diff
@@ -51,6 +51,9 @@ const setupToken = ({
     if (value.includes('px')) {
       return parseInt(value, 10);
     }
+    if (path[0] === 'borderWidths' || path[0] === 'space') {
+      return parseFloat(value);
+    }
```

Comment:
To make the logic in `setupToken` a bit easier to reason about in the future, what do you think abt combining the conditional block here the block immediately below and the block on line 43 and abstracting a util for checking whether the value should be passed to `parseFloat` ? e.g. 
```
// contrived name
const shouldParseFloatValue = (pathKey: string) => pathKey === 'space' || pathKey === 'borderWidths' || pathKey === 'opacities';

const setupToken = (
  // ...setupToken params
) {
  // ...other setupToken logic

  if (shouldParseFloat(path[0])) {
    if (value.includes('rem')) {
      return Math.floor(parseFloat(value) * 16 * spaceModifier);
    }
  
    return parseFloat(value);
  }
}